### PR TITLE
Messaging banner for T1 devices with older than 1.7.0 FW

### PR DESCRIPTION
**Pull request by MiroslavProchazka**
Resolving https://github.com/trezor/trezor-suite/issues/12624

### CODE REVIEW

**Comment by MiroslavProchazka** [comment]
I had to adjust the file several times to pass the message-system validate config, so the section "Files changed" might not be the best place to check the PR. Please checkout to `Node-bridge-transition-messaging` verify the code.

File: suite-common/message-system/config/config.v1.json [hunk 1]
```diff
@@ -795,44 +795,23 @@
...
-                        "web": ">=22.6.2"
+                        "web": ">=21.7.1"
                     },
                     "devices": [
                         {
                             "model": "1",
-                            "firmware": "<1.11.1",
+                            "firmware": "<1.7.0",
                             "bootloader": "*",
                             "variant": "regular",
```
Comment (matejkriz) [inline suite-common/message-system/config/config.v1.json]: variant should be `*` (affects also bitcoin-only FW...)

File: suite-common/message-system/config/config.v1.json [hunk 2]
```diff
@@ -795,44 +795,23 @@
...
-                        {
-                            "zec": true
-                        }
-                    ],
                     "environment": {
-                        "desktop": ">=22.6.2",
+                        "desktop": ">=21.7.1",
                         "mobile": "!",
-                        "web": ">=22.6.2"
+                        "web": ">=21.7.1"
```
Comment (matejkriz) [inline suite-common/message-system/config/config.v1.json]: why from 21.7.1 Suite version? Is it the version where FW 1.7.0 has been introduced?

**Comment by matejkriz** [comment]
Please add it in https://www.notion.so/satoshilabs/Secure-message-system-5ade512cbd38481f9d5f65f03dbfd9a8?pvs=4 🙏

File: suite-common/message-system/config/config.v1.json (hunk 2 above)
Comment (MiroslavProchazka) [reply suite-common/message-system/config/config.v1.json]: It is one of the first Suite released, so I wanted to make sure that practically everyone will receive it

Comment (matejkriz) [reply suite-common/message-system/config/config.v1.json]: Then you can use `*` for any version or remove that environment condition completely.

**Review by matejkriz — APPROVED** [review]
Consider removing the environment condition, but otherwise it looks good to me and it can stay as it is. I'm not sure what would internal-link do in mobile app, so it's safer to keep it excluded, or test it.

**Pull request merged**